### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by radek1st**
unit tests and coverage report

### CODE REVIEW

**Review by Arachnid — APPROVED** [review]

File: test/test_metamultisig.js [hunk 1]
```diff
@@ -1,53 +1,439 @@
...
+        try {
+            await multisig.submit(keyHolder3, web3.utils.toWei('2.0', 'ether'), '0x', nonce, sigs, {from: nonKeyHolder});
+            assert.fail("Expected exception");
+        } catch (e) {
+            assert.ok(e.message.includes('revert'));
+        }
+        assert.equal(await web3.eth.getBalance(multisig.address), web3.utils.toWei('1.0', 'ether'));
+    });
+
+    it('rejects already executed transaction, invalid nonce', async function () {
```
[the remark this answers is not in the document]
Comment (radek1st) [reply test/test_metamultisig.js]: added

File: contracts/MetaMultisig.sol [hunk 2]
```diff
@@ -49,16 +49,18 @@ contract MetaMultisig {
 
     function setKeyholderWeight(address keyholder, uint weight) external selfOnly {
         totalWeight = totalWeight.sub(keyholders[keyholder]).add(weight);
-        keyholders[keyholder] = weight;
```
[the remark this answers is not in the document]
Comment (radek1st) [reply contracts/MetaMultisig.sol]: reads better :)

File: test/test_metamultisig.js [hunk 3]
```diff
@@ -1,53 +1,439 @@
...
+            assert.fail("Expected exception");
+        } catch (e) {
+            assert.ok(e.message.includes('revert'));
+        }
+    });
+
+    it('sends tokens from multisig', async function () {
+        const nonce = await multisig.nextNonce();
+
+        const data = await web3.eth.abi.encodeFunctionCall({
```
[the remark this answers is not in the document]
Comment (radek1st) [reply test/test_metamultisig.js]: I'll leave it for next time, thanks

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# MetaMultisig
+
+
+## Unit tests
+
+To run unit tests with Truffle:
+
+```
+truffle develop
```
[the remark this answers is not in the document]
Comment (radek1st) [reply README.md]: `npm run coverage`

**Pull request merged**